indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -180,6 +180,13 @@
   (or (not (empty? (:DirectDistributionInformation collection)))
       (tag/has-cloud-s3-tag? tags)))
 
+(defn- standard-product?
+  "Test if the collection meets the criteria for being standard-product"
+  [collection tags]
+  (or (= true (:StandardProduct collection))
+      (and (not (:StandardProduct collection))
+           (tag/has-standard-product-tag? tags))))
```

Comment:
You don't need (and (not (:StandardProduct collection)).
Just
(or (= true (:StandardProduct collection))
      (tag/has-standard-product-tag? tags)))

Reply:
I did it because you mentioned that when StandardProduct is false, we index it as false.

Reply:
Fixed!